fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -159,8 +159,9 @@ public void setLastRefreshTime(long lastRefreshTime) {
     @SerializedName(value = "refreshScheme")
     private MvRefreshScheme refreshScheme;
 
-    @SerializedName(value = "baseTableIds")
-    private Set<Long> baseTableIds;
+    // if table has alias, use alias instead name
+    @SerializedName(value = "baseTableIdNames")
+    private Map<Long, String> baseTableIdNames;
```

Comment:
why we need to store base table alias in MV definition？

Reply:
I used  it to get  the table which partitions mapped with mv partitions in class `MvTaskRunProcessor` 
 ```
OlapTable partitionTable = null;
for (Long tableId : baseTableIdNames.keySet()) {
    OlapTable olapTable = (OlapTable) database.getTable(tableId);
    if (slotRef.getTblNameWithoutAnalyzed().getTbl().equals(baseTableIdNames.get(tableId))) {
        partitionTable = olapTable;
    }
    olapTables.put(olapTable.getId(), olapTable);
}
```